tests/ha/test_multi_server_pod_failure.py

```diff
@@ -629,3 +630,484 @@ def test_writes_till_kserver_pods_fail(self):
+                    self.kvalue)
+
+        LOGGER.info("Step 8: Check services status that were running on server pods "
+                    "which are deleted.")
+        counter = 0
+        for pod_name in self.pod_name_list:
+            hostname = self.pod_dict.get(pod_name)[0]
+            resp = self.hlth_master_list[0].get_pod_svc_status(pod_list=[pod_name], fail=True,
+                                                               hostname=hostname)
+            LOGGER.debug("Services status on %s : %s", pod_name, resp)
+            if not resp[0]:
+                counter += 1
+            pod_list.remove(pod_name)
+        assert_utils.assert_equal(counter, 0, "Services on some server pods not stopped.")
+        LOGGER.info("Step 8: Services of server pods are in offline state")
+
+        LOGGER.info("Step 9: Check services status on remaining pods %s", server_data_pod_list)
+        resp = self.hlth_master_list[0].get_pod_svc_status(pod_list=server_data_pod_list,
+                                                           fail=False)
+        LOGGER.debug("Response: %s", resp)
+        assert_utils.assert_true(resp[0], resp)
+        LOGGER.info("Step 9: Services of remaining pods are in online state")
+
+        LOGGER.info("Step 10: Verify status for In-flight READs/WRITEs/DELETEs while "
+                    "%s (K) server pods were going down.", self.kvalue)
+        LOGGER.info("Waiting for background IOs thread to join")
+        thread_wri.join()
+        thread_rd.join()
+        thread_del.join()
+        LOGGER.info("Step 10.1: Verify status for In-flight DELETEs while %s (K) server pods were"
+                    "going down", self.kvalue)
+        del_resp = tuple()
+        while len(del_resp) != 2:
+            del_resp = del_output.get(timeout=HA_CFG["common_params"]["60sec_delay"])
+        if not del_resp:
+            assert_utils.assert_true(False, "Background process failed to do deletes")
+        fail_del_bkt = del_resp[1]
+        rem_bkts_aftr_del = s3_test_obj.bucket_list()[1]
+        assert_utils.assert_false(len(fail_del_bkt),
+                                  f"Bucket deletion failed when cluster was online {fail_del_bkt}")
+        assert_utils.assert_true(len(rem_bkts_aftr_del) < del_bucket,
+                                 "Some bucket deletion expected during pods going down")
+        LOGGER.info("Step 10.1: Verified status for In-flight DELETEs while %s (K) server"
+                    " pods were going down", self.kvalue)
+
+        LOGGER.info("Step 10.2: Verify status for In-flight WRITEs while %s (K) server pods going "
+                    "down should be failed/error.", self.kvalue)
+        responses_wr = dict()
+        while len(responses_wr) != 2:
+            responses_wr = output_wr.get(timeout=HA_CFG["common_params"]["60sec_delay"])
+        pass_logs = list(x[1] for x in responses_wr["pass_res"])
```

Comment:
Please add debug statement for these logs
LOGGER.debug("Pass logs list: %s", pass_logs), this was observed helpful in many tests. Please take care where needed